### ISSUE TITLE
gradient accumulation

### ISSUE THREAD

**Issue by HardLaugh** [open-mmlab/mmdetection#1950]
what should i do to achieve gradient accumulation. thanks very much

**Comment by hellock** [comment]
Could you be more specifc? You want to forward multiple times and then do backpropagation once?

**Comment by HardLaugh** [comment]
@hellock you are right. I want to forward multiple times for the purpose of increasing batchsize

**Comment by hellock** [comment]
[Here](https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/utils/dist_utils.py#L53) is where backpropagation is done, and you can modify it as you want.
